### PR TITLE
Upper lower

### PR DESCRIPTION
**Pull request by chang111**
To fix the issue 3248

### CODE REVIEW

**Comment by chang111** [comment]
@junpenglao All check has passed. Please review it.

File: pymc3/distributions/continuous.py [hunk 1]
```diff
@@ -571,8 +571,8 @@ def __init__(self, mu=0, sigma=None, tau=None, lower=None, upper=None,
         tau, sigma = get_tau_sigma(tau=tau, sigma=sigma)
         self.sigma = self.sd = tt.as_tensor_variable(sigma)
         self.tau = tt.as_tensor_variable(tau)
-        self.lower = tt.as_tensor_variable(floatX(lower)) if lower is not None else lower
-        self.upper = tt.as_tensor_variable(floatX(upper)) if upper is not None else upper
+        self.lower = tt.as_tensor_variable(floatX(lower)) if lower is not None else tt.as_tensor_variable(-np.inf)
```
Comment (lucianopaz) [inline pymc3/distributions/continuous.py:574]: You have to also `floatX` `np.inf`

**Review by lucianopaz — CHANGES_REQUESTED** [review]
The commit history is a bit weird and shows all the commits you had done in your previous PR.

Nevertheless, you need to do 3 more things to get this done:
1. Add a line to the 3.7 release notes under maintenance specifying this fix.
2. Add tests to the test suite to explicitly test and ensure that the issue, #3248, is fixed and all potential values for the bounds work well.
3. Be careful about the `_normalization` function that relies on the bounds being `None` or not. You should change this to some `isinf` kind of control.

There was a previous PR, #3250, that did this but became stale and was never finished. You can look at it to get some ideas.

File: pymc3/distributions/continuous.py (hunk 1 above)
Comment (chang111) [reply pymc3/distributions/continuous.py:574]: Thanks for your reply. I will check the pr 3250.

**Comment by twiecki** [comment]
@chang111 Did you mean to close this?

**Pull request closed without merging**